docs/DEVELOPMENT.md

```diff
@@ -64,8 +64,9 @@ API development uses [Goa code generator](https://goa.design/).
 
 To run the API locally:
 1. Start `bash` in the dev container, run `docker-compose run --rm -u "$UID:$GID" --service-ports dev bash`.
-2. In the container run `make run-templates-api`
- - The API is exposed to `http://localhost:8000/`.
+2. Set env var with Keboola stack: `TEMPLATES_API_STORAGE_API_HOST=connection.keboola.com` (or `BUFFER_API_STORAGE_API_HOST=connection.keboola.com`)
```

Comment:
Asi by bylo lepší, kdyby bylo možný v tom airu nastavit CLI argument a předat do něj env var (sem: https://github.com/keboola/keboola-as-code/blob/7999495be19e16533637c62a3387395c28565907/provisioning/buffer/dev/.air-api.toml#L6-L6), ale nepřišel jsem na to jak to udělat. Problém je že ten konfig není go file a musí to podporovat přímo ta toola. (Tady se řešilo něco podobnýho, ale to je o nastavení a předání env vars dál do spuštěné binárky: https://github.com/cosmtrek/air/issues/58 my potřebujeme vzít existující env var a injectnout ji do toho konfigu)

Reply:
Kazdy flag sa da zadat ako ENV, v tomto pripade `BUFFER_API_LISTEN_ADDRESS`, ... tj. nemusis to asi nutne dostat do configu, staci, ze bude nastavana dana ENV (docker-compose) a `air` ju posunie dalej (?) - alebo to takto nejde?

Neviem presne co chces dosiahnut, chces sa zbavit statickej hodnoty v tomto configu?

Reply:
Jo jo vím, ale právě ten `air` nepředá env vars dál. Tak jsem to chtěl dostat z env var do konfigu (jako `args_bin = ["--listen-address=http://0.0.0.0:8001", "--public-address=http://localhost:8001", "--storage-api-host=$BUFFER_API_STORAGE_API_HOST"]`). Zdá se ale že nejde ani jedna varianta.

Podle toho https://github.com/cosmtrek/air/issues/58 se mi to podařilo rozjet jen se statickou hodnotou když přidám do konfigu `full_bin = "BUFFER_API_STORAGE_API_HOST=connection.keboola.com ./target/buffer/api"`

Reply:
Asi to potom nechajme takto, nemam aktualne kapacitu to riesit a nepride mi to ako priorita.

Reply:
jasně, bylo by to jen pěknější. 🙂